packages/standard-action-handler/src/types.ts

```diff
@@ -6,12 +6,21 @@ export interface NavigateOptions {
   [key: string]: unknown
 }
 
+export type TargetType = 'current' | 'new' | 'browser'
```

Comment:
해당 타입은 `ContextOptions`에서만 사용하는 걸로 보여요 👀   
export를 제거해도 되지 않을까요?

```suggestion
type TargetType = 'current' | 'new' | 'browser'
```